ArcGISRuntimeSDKQt_CppSamples/Analysis/LineOfSightLocation/README.md

```diff
@@ -1,17 +1,32 @@
 # Line of sight (location)
 
-This sample demonstrates how to perform a line of sight analysis between two points in a SceneView.
+Perform a line of sight analysis between two points in real time.
 
 ![](screenshot.png)
 
+## Use case
+
+A line of sight analysis can be used to assess whether a view is obstructed between an observer and a target. Obstructing features could either be natural, like topography, or man-made, like buildings. Consider an events planning company wanting to commemorate a national event by lighting sequential beacons across hill summits or roof tops. To guarantee a successful event, ensuring an unobstructed line of sight between neighboring beacons would allow each beacon to be activated as intended.
+
+## How to use the sample
+
+The sample loads with a preset observer and target location, linked by a colored line. A red segment on the line means the view between observer and target is obstructed, whereas green means the view is unobstructed. 
+
+Click to turn on the mouse move event listener, then move the mouse to update the target location in real time. Click again to lock the target location.
```

Comment:
I think this might be a slight difference between Java and Qt. For Qt, I believe you do a press and hold and then drag around the scene instead of a click to start and then hover